release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1399,6 +1454,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * The ability to delete operands from the web console using the `delete all operands` option is currently disabled. It will be re-enabled in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2012120[BZ#2012120] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2012971[BZ#2012971].
 
+* This release contains a known issue with Jenkins. If you customize the hostname and certificate of the OpenShift OAuth route, Jenkins no longer trusts the OAuth server endpoint. As a result, users cannot log in to the Jenkins console if they rely on the OpenShift OAuth integration to manage identity and access.
```

Comment:
We should also include https://bugzilla.redhat.com/show_bug.cgi?id=2055653 as a known issue. This is something that we plan on fixing in an early z-stream. It was found late and didn't make the release cutoff.

Reply:
Thanks. I've added it below, on line 1521.